APISpecification.yaml

```diff
@@ -91,6 +109,18 @@ paths:
                 type: array
                 items:
                   $ref: '#/components/schemas/ExampleObject'
+        '404':
+          description: Genome not found
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: "Could not find for a7335667-93e7-11ec-a39d-005056b38ce3"
```

Comment:
> Could not find for

Find what?

Reply:
That's how the code corrently implements the error message. I just updated the spec to reflect that (like for the rest of the endpoints). I can update the code in this for something better in this PR if it makes sense.

Reply:
Code and spec updated.